src/components/shared/chapter-page.tsx

```
@@ -22,40 +22,58 @@ export const ChapterPage = async ({
-      :(
-        <article className='space-y-2 prose lg:prose-xl dark:prose-invert max-w-none' dangerouslySetInnerHTML={{__html: chapter.content}} />
-      )}
-      <div className='flex justify-between pt-12 pb-4'>
+    <div className='p-4 space-y-6'>
+      {/* Reading container with neobrutalism styling */}
+      <div className='max-w-4xl mx-auto bg-background md:border-brutal md:border-black md:dark:border-white rounded-lg md:shadow-brutal-lg p-2 md:p-6 md:p-12'>
+        <H3 className='border-b-4 border-brutal-cyan pb-2 mb-6'>Chapter {chapter.number}: {chapter.title}</H3>
+        {(chapter.premium) ? 
+        <>
+        <SignedIn>
+          <article className='space-y-2 prose lg:prose-xl dark:prose-invert max-w-none chapter-content' dangerouslySetInnerHTML={{__html: chapter.content}} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)